README.md

```diff
@@ -4,3 +4,21 @@ Docker files to build dockers based on:
 * OCB code
 
 This work is original based on the docker from http://www.xcg-consulting.fr and available here: https://hub.docker.com/r/xcgd/odoo/
+
+oca_dependencies.txt
+
```

Comment:
@elicoidal No , I don't know how to write the introduction , It's a bit hard for me to write a formal documentation like this.